server/data/interfaces/prisonApiClient.ts

```diff
@@ -102,7 +102,7 @@ export interface PrisonApiClient {
   getAgencyDetails(agencyId: string): Promise<AgencyLocationDetails>
   getOffenderCellHistory(bookingId: number, params: object): Promise<OffenderCellHistory>
   getReceptionsWithCapacity(agencyId: string, attribute?: string): Promise<Reception[]>
-  getStaffDetails(staffId: string): Promise<StaffDetails>
+  getStaffDetails(username: string): Promise<StaffDetails>
```

Comment:
This was incorrectly named, it's actually the username that's being used to make the API call